.evergreen/auth_oidc/oidc_get_tokens.py

```diff
@@ -4,11 +4,20 @@
 HERE = os.path.abspath(os.path.dirname(__file__))
 sys.path.insert(0, HERE)
 from utils import get_secrets, get_id_token, DEFAULT_CLIENT, join
+TOKEN_DIR = os.environ['OIDC_TOKEN_DIR'].replace(os.sep, '/')
+
+def generate_tokens(config, base_name):
+    os.makedirs(TOKEN_DIR, exist_ok=True)
+    config['token_file'] = join(TOKEN_DIR, base_name)
+    get_id_token(config)
+    for i in range(2):
```

Comment:
Is there a future case where we might have more than 2 users? Could we use a defined constant instead of a magic number throughout this file?

Reply:
This value is for the number of extra tokens for a given user.